roles/backups/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- include_role:
+    name: coopdevs.backups_role
+    tasks_from: main.yml
+  vars:
+    backups_role_script_prepare_template: "db_and_filesystem_backup.sh.j2"
+    backups_role_db_name: "{{ database_name }}"
+    backups_role_assets_paths: ["{{ backups_role_tmp_path }}/backup.tar.gz"]
```

Comment:
this should be a diferent var named something like `backups_role_file_to_upload` or `backups_role_resulting_backup_file`. Now it's a var used with two different purposes => :boom:

Reply:
I documented this and more in https://github.com/coopdevs/backups_role/pull/40